provider_sources/leaflet-providers-parsed.json

```diff
@@ -0,0 +1 @@
ttribution": "![](https://docs.onemap.sg/maps/images/oneMap64-01.png) New OneMap | Map data (C) contributors, Singapore Land Authority", "name": "OneMapSG.LandLot"}}, "USGS": {"USTopo": {"url": "https://basemap.nationalmap.gov/arcgis/rest/services/USGSTopo/MapServer/tile/{z}/{y}/{x}", "max_zoom": 20, "attribution": "Tiles courtesy of the U.S. Geological Survey", "name": "USGS.USTopo"}, "USImagery": {"url": "https://basemap.nationalmap.gov/arcgis/rest/services/USGSImageryOnly/MapServer/tile/{z}/{y}/{x}", "max_zoom": 20, "attribution": "Tiles courtesy of the U.S. Geological Survey", "name": "USGS.USImagery"}, "USImageryTopo": {"url": "https://basemap.nationalmap.gov/arcgis/rest/services/USGSImageryTopo/MapServer/tile/{z}/{y}/{x}", "max_zoom": 20, "attribution": "Tiles courtesy of the U.S. Geological Survey", "name": "USGS.USImageryTopo"}}, "WaymarkedTrails": {"hiking": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "hiking", "name": "WaymarkedTrails.hiking"}, "cycling": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "cycling", "name": "WaymarkedTrails.cycling"}, "mtb": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "mtb", "name": "WaymarkedTrails.mtb"}, "slopes": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "slopes", "name": "WaymarkedTrails.slopes"}, "riding": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "riding", "name": "WaymarkedTrails.riding"}, "skating": {"url": "https://tile.waymarkedtrails.org/{variant}/{z}/{x}/{y}.png", "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors | Map style: (C) waymarkedtrails.org (CC-BY-SA)", "variant": "skating", "name": "WaymarkedTrails.skating"}}, "OpenAIP": {"url": "http://{s}.tile.maps.openaip.net/geowebcache/service/tms/1.0.0/openaip_basemap@EPSG%3A900913@png/{z}/{x}/{y}.{ext}", "attribution": "openAIP Data (CC-BY-NC-SA)", "ext": "png", "min_zoom": 4, "max_zoom": 14, "tms": true, "detectRetina": true, "subdomains": "12", "name": "OpenAIP"}, "OpenSnowMap": {"pistes": {"url": "https://tiles.opensnowmap.org/{variant}/{z}/{x}/{y}.png", "min_zoom": 9, "max_zoom": 18, "attribution": "Map data: (C) OpenStreetMap contributors & ODbL, (C) www.opensnowmap.org CC-BY-SA", "variant": "pistes", "name": "OpenSnowMap.pistes"}}}
```

Comment:
Could we "prettify" the json we include? (unless the compact form is significantly more efficient or so)

Reply:
Yes, it is on the list. See https://github.com/geopandas/xyzservices/issues/9 for a bit of relevant discussion.